.github/workflows/test-bytecode-parser.yml

```diff
@@ -0,0 +1,49 @@
+name: Test Bytecode Parser
+
+on:
+  pull_request:
+    paths:
+      - py-polars/**
+      - .github/workflows/test-bytecode-parser.yml
+  push:
+    branches:
+      - main
+    paths:
+      - py-polars/**
+      - .github/workflows/test-bytecode-parser.yml
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: py-polars
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
What do you think about getting ahead of the game and adding "3.12-dev" too? :)

Reply:
great idea, but looks like numpy fails to install there